src/views/navbar.hbs

```diff
@@ -0,0 +1,223 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="/css/navbar.css">
+<link rel="stylesheet" href="/css/style.css">
```

Comment:
It is very wrong for you to call `link` or `script` tags inside your body, the `link`s belong to the `<head></head>` and the scripts are appended to the end of the body. Figure a way to do that correctly. Do not call links nor scripts in hbs pages.